tools/highlight-files/highlight-files.ts

```diff
@@ -25,21 +26,48 @@ function getBazelActionArguments() {
   return args;
 }
 
+function _detectAndHighlightRegionBlocks(parsed:
```

Comment:
I don't think this has to be named with an `_` since its just a random node script, not something that'll show up in our API